app/code/Magento/Config/view/adminhtml/templates/system/config/tabs.phtml

```diff
@@ -4,11 +4,14 @@
  * See COPYING.txt for license details.
  */
 
-/** @var $block \Magento\Config\Block\System\Config\Tabs */
+/**
+ * @var $block \Magento\Config\Block\System\Config\Tabs
+ * @var \Magento\Framework\Escaper $escaper
+ */
 ?>
 
```

Comment:
It looks like we don't need to have an empty line here
```suggestion
```

Reply:
fixed